modules/driver-jdbc/src/main/kotlin/datamaintain/db/driver/jdbc/JdbcDriver.kt

```diff
@@ -3,14 +3,15 @@ package datamaintain.db.driver.jdbc
 import datamaintain.core.db.driver.DatamaintainDriver
 import datamaintain.core.script.*
 import datamaintain.core.step.executor.Execution
+import datamaintain.db.driver.jdbc.exception.JdbcQueryException
 import java.sql.Connection
 import java.sql.DriverManager
 import java.sql.ResultSet
 import java.sql.SQLException
 import java.util.*
 
-class JdbcDriver(jdbcUri: String) : DatamaintainDriver(jdbcUri) {
-    private val connection: Connection = DriverManager.getConnection(jdbcUri)
+class JdbcDriver(jdbcUri: String,
+                 private val connection: Connection = DriverManager.getConnection(jdbcUri)) : DatamaintainDriver(jdbcUri) {
```

Comment:
Giving the connection as attribute enables the user to provide a connection to another database than the one corresponding to the URI

Reply:
But it helps me perform some test by giving my own connection